content/en-US/api/fontStretch.md

```diff
@@ -0,0 +1,68 @@
+---
+recipe: api-interface
+title: 'CanvasRenderingContext2D.fontStretch'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/fontStretch
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.CanvasRenderingContext2D.fontStretch
+---
+
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/textLetterSpacing](https://developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/textLetterSpacing).**
+
+## Description
+
+The `fontStretch` property of the `CanvasRenderingContext2D` interface
+allows the developers to select different face from a font. The default
```

Comment:
I may be wrong, but I think it's unusual to see the word "face" alone in this context. i.e. I think this should read:
"developers to select different font-face from a font."

Reply:
it think yours reads better. Update it.

Reply:
It didn't seem strange to me. 

By the way, it should be "font face" (no hyphen) not "font-face". Please change.